src/Makefile

```diff
@@ -49,7 +49,7 @@ SC_VER_STR	:= $(SC_VER_MAJ).$(SC_VER_MIN).$(SC_VER_REL).$(SC_VER_BUILD)
 # by SC_API_VER_MAX in src/include/solar_capture_ext.h.
 _VER_INC := -I${SRCDIR}/include
 SCLIB_VER_MAJOR := ${SC_VER_MAJ}
-SCLIB_VER_MINOR := $(shell { echo "\#include \"solar_capture_ext.h\""; echo "SC_API_VER_MAX"; } | cpp ${_VER_INC} | tail -1 )
```

Comment:
I want to draw attention to the GNU make's backwards-incompatibility around the `#` sign. Removing the escape might upset older versions (I guess 3.81 and older): https://lists.gnu.org/archive/html/info-gnu/2020-01/msg00004.html.

@rgarner-vivcourt Does this line cause any build-time issues? If not, shall we keep it as-is? Otherwise, shall we apply the portability suggestion from the release notes (`H := \#`)?

Reply:
The build failed on our RHEL 9.2 system with the original Makefile - using the `H := \#` fix sounds like a better approach.